js/apps/system/_admin/aardvark/APP/react/src/views/collections/indices/listIndices/CollectionIndicesTable.tsx

```diff
@@ -18,16 +18,25 @@ const NameCell = ({ info }: { info: CellContext<Index, string> }) => {
   const collectionName = window.location.hash.split("#cIndices/")[1]; // get the collection name from the url
   // need to use href here instead of RouteLink due to a bug in react-router
   return (
-    <Link
-      href={`#cIndices/${collectionName}/${finalId}`}
-      textDecoration="underline"
-      color="blue.500"
-      _hover={{
-        color: "blue.600"
-      }}
-    >
-      {info.cell.getValue()}
-    </Link>
+    <Text color="gray.400">
```

Comment:
as discussed, let's change this color to be darker, and check if the index is still clickable –– if so we'd need to show progress inside the screen too. If not, we should disable the row/link click